src/relay/op/image/resize.cc

```diff
@@ -33,6 +33,23 @@ namespace relay {
 
 TVM_REGISTER_NODE_TYPE(ResizeAttrs);
 
+Array<Array<Layout> > ResizeInferCorrectLayout(const Attrs& attrs,
+                                               const Array<Layout>& new_in_layouts,
+                                               const Array<Layout>& old_in_layouts,
+                                               const Array<tvm::relay::Type>& old_in_types) {
+  // NOTE: Discard "const" qualifier here.
+  ResizeAttrs* params = const_cast<ResizeAttrs*>(attrs.as<ResizeAttrs>());
+
+  if (new_in_layouts.defined()) {
```

Comment:
When I was working on the `transpose`, I found that it's possible to have `new_in_layouts` defined but `new_in_layouts[0]` undefined. I'm not sure if it's a case in `resize` tho.